packages/flutter_tools/lib/src/android/multidex.dart

```diff
@@ -93,9 +93,7 @@ bool androidManifestHasNameVariable(final Directory projectDir) {
   XmlDocument document;
   try {
     document = XmlDocument.parse(manifestFile.readAsStringSync());
-  } on XmlParserException {
-    return false;
-  } on XmlTagException {
```

Comment:
Here I had previously added additional handling for XmlTagException, however this wouldn't have caught an `XmlParentException` or `XmlNodeTypeException`